### ISSUE TITLE
Feature Request: Alternative English Audio

### ISSUE THREAD

**Issue by andrew-dixon** [leonardoxh/race-control-tv#111]
On the F1TV app on the iPad there are two different English audio feeds, the "world feed" (bottom option) and the F1TV feed top option.

![IMG_FCB2E366F7C8-1](https://user-images.githubusercontent.com/7441801/160202309-0936fddc-e2ec-4453-a503-a66a8998f8da.jpeg)

ATM the app only provides the "world feed" audio, e.g. the UK Sky Sports F1 commentary, would it be possible to make the F1TV audio available as well (mainly to stop having to listen to David Croft 😂 ).

**Comment by Pkshields** [comment]
+1. Last week the F1 Live commentary was available on the old pit lane channel button, but for qualifying this week it doesn't seem to be available on any stream. Using the official F1 app now and it's overactive adaptive streaming app is driving me crazy, would greatly appreciate the F1 Live commentary in this app.

**Comment by leonardoxh** [comment]
Yeah, well, I think the 2nd audio option is in the `pit lane` channel... So we would have to merge these 2 channels somehow, doable, but it is a lot of work if you ask me.

**Comment by andrew-dixon** [comment]
@leonardoxh The pit channel is a separate option in the app, I'm not sure if it is the same audio, probably, but I had assumed the second "English" audio would be available as an option like the other languages are in the main feed, is that not the case then?

**Comment by andrew-dixon** [comment]
Also, in the F1TV app, the default audio is now the F1TV commentary., not sure if that makes any difference.

**Comment by andrew-dixon** [comment]
Ok, So I've just been looking on the app and noticed that for 2022 the "pit lane channel" no longer exists in the F1TV app:

2021:
![IMG_0131](https://user-images.githubusercontent.com/7441801/160346127-4cbde834-22a8-4fc2-9f02-7109ef076a14.jpg)

2022:
![IMG_0132](https://user-images.githubusercontent.com/7441801/160346153-eb09b6db-4ae1-4edb-a133-87357129fa16.jpg)

On the. 2022 version, the top two options provide the same video, but different audio, but on the 2021 version, the top two options provide different audio and video.

**Comment by leonardoxh** [comment]
Yeah they are different videos sources in fact, but they somehow mix them up.

It shouldn't be that hard to put them together, it is just a lot of work.

**Comment by andrew-dixon** [comment]
So last year on your app you had "Main" and "Pit Lane" channel options, but this year there is only the "Main" option with the Sky Sports F1 audio feed. Would it not be possible to add a second option for the "F1TV" feed if they are two different video sources? Surely that isn't much work as you already had it previously. Maybe I'm misunderstanding what you are saying.

**Comment by leonardoxh** [comment]
Ehm,

What I mean is that if you take a look at the channel selection you'll see the pit lane channel next to the main channel, the audio you're looking for is in this pit lane channel, which is the same as the main channel but with only English available...

That is the magic on the official app, they use both channels at same time

**Comment by andrew-dixon** [comment]
@leonardoxh - Not in 2022. Take a look, there is no longer a pit lane channel, it's gone from both the official app and your app. If you look at a 2021 race, the pit lane channel is there, but for 2022 races, it's no longer there. The screenshot in my comment above shows this. The "security camera" icon that was the pitlane channel before has gone.

**Comment by leonardoxh** [comment]
Ah I see, for the first race is available, but now for the second race isn't available anymore.

I had the impression that it was available last weekend.

Needs some exploration then

**Comment by andrew-dixon** [comment]
There is some discussion of the 2022 changes here: https://www.reddit.com/r/F1TV/comments/thomkv/the_state_of_f1tv_march_2022/

**Comment by LoVega1337** [comment]
I have created a possible fix in my [Pull Request here](https://github.com/leonardoxh/race-control-tv/pull/117), might need some testing tho, don't have Android Studio installed currently.

**Comment by leonardoxh** [comment]
Ah like that, I will check it this weekend

**Comment by andrew-dixon** [comment]
@leonardoxh @LoVega1337 I noticed this morning in during FP2 they seemed to have different video feeds on the two channels this weekend... so I guess they are developing it over time.

**Comment by LoVega1337** [comment]
@andrew-dixon Hmm which video feeds are you talking about? WIF and International (if they are even seperated) or the International one and F1Live? If it's the latter, it is supposed to be like that since F1TV started to do an own casting team with their own "Feed" in the F1Live Channel.

**Comment by andrew-dixon** [comment]
@LoVega1337 As described above, on the F1TV app, for the first two races the video feeds on the two options had the same video output, just different audio, however, today I noticed during FP2 on the "default" feed they were using a "two box" set up at points with the world feed in one and something else in the other, e.g. an interview, however switching to the second option (the one that looks like a globe), it was still just the world feed (not two box) with the Sky Sports commentary. I guess maybe this was always the case "under the hood", they just hadn't got to using a different video output until this weekends race. Looking at your code changes, it seems this should probably resolve this and allow the selection of each of the feeds. 

@leonardoxh any chance of making a binary available with @LoVega1337 pull request in it to test this out over the weekend?

**Comment by LoVega1337** [comment]
> @LoVega1337 As described above, on the F1TV app, for the first two races the video feeds on the two options had the same video output, just different audio, however, today I noticed during FP2 on the "default" feed they were using a "two box" set up at points with the world feed in one and something else in the other, e.g. an interview, however switching to the second option (the one that looks like a globe), it was still just the world feed (not two box) with the Sky Sports commentary. I guess maybe this was always the case "under the hood", they just hadn't got to using a different video output until this weekends race. Looking at your code changes, it seems this should probably resolve this and allow the selection of each of the feeds.
> 
> @leonardoxh any chance of making a binary available with @LoVega1337 pull request in it to test this out over the weekend?

I see what you mean, yes this is exactly what I meant. The International one is the one with the Sky commentary, this was always there. The other one you described with the Splitscreen and stuff is the F1 Live channel, formerly known as the Pit Lane Channel. They were always seperate channels, but they used the Pit Lane Channel differently and rebranded it now to the F1 Live channel. Before Saudi Arabia they still used the backend name for the Pit Lane Channel to broadcast the new F1 Live channel, but since renamed that backend name to something new, that's why it doesn't show up currently.

tl;dr: With my fix all available channels (including the F1 Live channel, formerly the Pit Lane Channel) will be available again. :D

**Comment by LoVega1337** [comment]
You can easily check that in the player on https://f1tv.formula1.com/. If 1080p is available there, it should also be available in the app.

**Comment by andrew-dixon** [comment]
Seems to be on the FP2 replay 👍 

![Screenshot 2022-04-08 at 17 21 37](https://user-images.githubusercontent.com/7441801/162482596-f716673f-5283-4534-8f91-39c55ba40bc9.png)

**Comment by LoVega1337** [comment]
Awesome, then it will be available in the app. 😃

**Comment by LoVega1337** [comment]
@andrew-dixon I built an APK with the fix through GitHub Actions, find it in my comment in my Pull request. https://github.com/leonardoxh/race-control-tv/pull/117#issuecomment-1093469500
Would be awesome if you could check if it works. But try FP1 only for now, since FP2 still had some DRM active and wouldn't load, thats another problem thats currently going on with every 3rd party app. :/

**Comment by andrew-dixon** [comment]
@LoVega1337 Unfortunately I'm stuck on the login screen with it telling me my username and password is incorrect. I don't think this is related to your build as I've had it before, so I'll try again tomorrow.
